File: BotsSDK/korebot/src/main/java/kore/botssdk/fileupload/listeners/FileTokenListener.java
package kore.botssdk.fileupload.listeners;

import java.io.IOException;
import java.util.Hashtable;

public interface FileTokenListener {
	/**
	 * Callback for when filetoken received successfully 
	 */
	void fileTokenReceivedSuccessfully(Hashtable<String, String> hsh) throws IOException;
	
	/**
	 * Callback for when filetoken service unsuccessful 
	 */
	void fileTokenReceivedWithFailure(String errCode, String reason);
}

File: BotSdk/ui/src/main/java/com/kore/uploadfile/listeners/FileTokenListener.kt
package com.kore.uploadfile.listeners

import java.util.Hashtable

interface FileTokenListener {
    /**
     * Callback for when file token received successfully
     */
    fun fileTokenReceivedSuccessfully(hsh: Hashtable<String, String>)

    /**
     * Callback for when file token service unsuccessful
     */
    fun fileTokenReceivedWithFailure(errCode: String, reason: String)
}